ui/app/components/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -178,3 +197,25 @@ function addHexWEIsToRenderableFiat (aHexWEI, bHexWEI, convertedCurrency, conver
     partialRight(formatCurrency, [convertedCurrency]),
   )(aHexWEI, bHexWEI)
 }
+
+function getRenderableTimeEstimate (currentGasPrice, priceAndTimeEstimates) {
+  const gasPrices = priceAndTimeEstimates.map(({ gasprice }) => gasprice)
+  const estimatedTimes = priceAndTimeEstimates.map(({ expectedTime }) => expectedTime)
+
+  const closestLowerValueIndex = gasPrices.findIndex((e, i, a) => {
+    return e <= currentGasPrice && a[i + 1] >= currentGasPrice
+  })
+  const closestHigherValueIndex = gasPrices.findIndex((e, i, a) => {
+    return e > currentGasPrice
```

Comment:
Will this always exist?

Reply:
Yes. However, I can clean up this code with some methods from `gas-price-chart.utils`

Reply:
Code clean up done in 1f0cb6e